stellargraph/data/explorer.py

```diff
@@ -203,7 +203,7 @@ def _walk(self, rs, start_node, length):
         current_node = start_node
         for _ in range(length - 1):
             neighbours = self.neighbors(current_node)
-            if not neighbours:
+            if not len(neighbours):
```

Comment:
I don't think this needed to change? Or if it does, does that mean the behaviour of `graph.neighbors` has changed?

Reply:
Ah, yeah, it has: it now returns a `np.array`, instead of a `list` and so the implicit `bool(...)` call here hits:

```
>>> bool(np.array([1, 2]))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

I'll change it back to returning a `list` so we're doing minimal breakage for now (at least until #718).